p4runtime_sh/shell.py

```diff
@@ -2561,10 +2564,10 @@ def pipe_config(arg):
     return parser
 
 
-def setup(device_id=1, grpc_addr='localhost:9559', election_id=(1, 0), config=None):
+def setup(device_id=1, grpc_addr='localhost:9559', election_id=(1, 0), role_name=None, config=None):
```

Comment:
I see your recent changes. Note that this only matters for the arbitration message. In Write / Read requests, the `role` is a scalar string field, and there is no difference - from a Protobuf message serialization perspective - between "not setting the field" and "setting the field to the empty string".

However the current version has the advantage that p4runtime-shell can keep working with older P4Runtime versions so that's nice.